docs/configuration.md

```diff
@@ -3275,6 +3275,24 @@ With the above configuration it is possible to run the `SET_ACTIVE_SPOOL ID=1`
 command to set the currently tracked spool ID to `1`, and the `CLEAR_ACTIVE_SPOOL`
 to clear spool tracking (useful when unloading filament for example).
 
+
+### `[klipper_estimator]`
+
+Enables gcode processing through [klipper_estimator](https://github.com/Annex-Engineering/klipper_estimator)
+to update gcode estimated time.
+
+```ini
+# moonraker.conf
+
+[klipper-estimator]
```

Comment:
```suggestion
[klipper_estimator]
```

The rest seems to work like a charm. Thank you

Reply:
missed this one, thanks